iothub/service/src/FeedbackBatch.cs

```diff
@@ -8,13 +8,28 @@ namespace Microsoft.Azure.Devices
 
     /// <summary>
     /// Contains a batch of feedback records.
+    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub.
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// Date and time that indicates when the feedback message was received by the IoT hub.
+        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub.
```

Comment:
did you mean to have the statement "Feedback messages are delivery acknowledgments for messages sent to a device from IoT Hub." repeated for the properties?

Reply:
yes it is intentional.